File: VB/T183237/Form1.vb
Imports System.ComponentModel
Imports System.Drawing

Namespace T183237

    Public Partial Class Form1
        Inherits DevExpress.XtraEditors.XtraForm

        Public Sub New()
            InitializeComponent()
            pictureEdit1.Image = imageCollection1.Images(0)
        End Sub
    End Class
End Namespace

File: CS/T183237/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;

namespace T183237 {
    public partial class Form1 : DevExpress.XtraEditors.XtraForm {
        public Form1() {
            InitializeComponent();
            pictureEdit1.Image = imageCollection1.Images[0];
        }
    }
}
